WBooks/Screens/BookLibrary/BookLibraryViewModel.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  BookLibraryViewModel.swift
+//  WBooks
+//
+//  Created by Federico Agustin Diaz on 01/08/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class BookLibraryViewModel {
+    
+    var libraryItems: [Book] = [] {
+        didSet {
+            self.reloadViewClosure?()
+        }
+    }
+    
+    var numberOfCells: Int {
+        return libraryItems.count
+    }
+    
+    var heightOfCells: CGFloat {
+        return 105
+    }
+    
+    var reloadViewClosure: (() -> Void)?
+    
+    func getCellViewModel(at indexPath: IndexPath) -> Book {
```

Comment:
This function name is confusing. The name says view model but you are returning a Book. I think `getCellBook(at:)` would be ok.